File: sample/src/main/java/com/thoughtbot/expandablerecyclerview/sample/multicheck/MultiCheckGenre.java
package com.thoughtbot.expandablerecyclerview.sample.multicheck;

import com.thoughtbot.expandablecheckrecyclerview.models.MultiCheckExpandableGroup;
import com.thoughtbot.expandablerecyclerview.sample.Artist;
import java.util.List;

public class MultiCheckGenre extends MultiCheckExpandableGroup {

  private int iconResId;

  public MultiCheckGenre(String title, List<Artist> items, int iconResId) {
    super(title, items);
    this.iconResId = iconResId;
  }

  public int getIconResId() {
    return iconResId;
  }
}

File: sample/src/main/java/com/thoughtbot/expandablerecyclerview/sample/multicheck/MultiCheckGenre.kt
package com.thoughtbot.expandablerecyclerview.sample.multicheck

import com.thoughtbot.expandablecheckrecyclerview.models.MultiCheckExpandableGroup
import com.thoughtbot.expandablerecyclerview.sample.Artist

class MultiCheckGenre(title: String?, items: List<Artist?>?, val iconResId: Int) : MultiCheckExpandableGroup(title, items)